test/python/compiler/test_compiler.py

```diff
@@ -198,6 +197,7 @@ def test_parallel_compile(self):
         qc.measure(qr[5], cr[0])
         qlist = [qc for k in range(10)]
         with self.assertWarns(DeprecationWarning):
+            backend = Fake20QV1()
```

Comment:
I liked your comment from the previous file :)

```suggestion
            # Ignorable, because assemble and BackendV1 are deprecated
            backend = Fake20QV1()
```

Reply:
multi branch PRing is hard :P

Reply:
ill remove all these comments. They are obvious, even for future @1ucian0